app/views/about/Index.html.erb

```diff
@@ -0,0 +1,73 @@
+<link href ='app/assets/stylesheets/aboutme.scss' type='text/css' rel='stylesheet'>
+<div class="name">     
+    <img width='300px' height='300px' src= 'https://cdn.filestackcontent.com/AchUBPpbtR12UdA8r3ilwz/security=policy:eyJleHBpcnkiOjIyNDE3MzU4MTksImNhbGwiOlsicmVhZCIsImNvbnZlcnQiXSwiaGFuZGxlIjoiWHg5N2dVT1BSSXFjanJvWkZJcHgifQ==,signature:87dd7e34f59faed2f03dc50d44bfff58f2bce4dfc2de26718cadd7fdc7610e1a/cache=expiry:max/resize=w:600,h:600,fit:crop,align:faces/rotate=d:exif/Xx97gUOPRIqcjroZFIpx '/> 
```

Comment:
You're using inline styling here to set the width and height of this image. While this is not wrong, it's good to note that these values could also live within the CSS file.

Reply:
Don't know how that happened, fixed now